internal/queue/score/score.go

```diff
@@ -0,0 +1,98 @@
+package score
+
+import (
+	"time"
+
+	"github.com/takenet/deckard/internal/dtime"
+)
+
+// TODO: keeping these as variables to get their reference and use less memory. Is this really necessary?
+// TODO: benchmark this
+var Max float64 = 9007199254740992
+var Min float64 = 0
+var Undefined float64 = -1
+
+// GetPullMaxScore returns the upper threshold priority filter for a pull request.
+// If the score is zero, it returns nil.
+// If the score is greater than Max, it returns Max.
+// If the score is less than Min, it returns Min.
+// Otherwise, it returns the score.
+func GetPullMaxScore(score float64) *float64 {
+	if score == 0 {
+		return nil
+	}
+
+	if score > Max {
```

Comment:
Can it really assume a value greater than max without causing an overflow ?

Reply:
Yes, check my answer for the other conversation.